src/dialects/postgres/query-generator.js

```diff
@@ -913,19 +1076,18 @@ export class PostgresQueryGenerator extends AbstractQueryGenerator {
    * @param {string} schemaName
    */
   getForeignKeyReferencesQuery(tableName, catalogName, schemaName) {
-    return `${this._getForeignKeyReferencesQueryPrefix()
-    }WHERE constraint_type = 'FOREIGN KEY' AND tc.table_name = '${tableName}'${
-      catalogName ? ` AND tc.table_catalog = '${catalogName}'` : ''
-    }${schemaName ? ` AND tc.table_schema = '${schemaName}'` : ''}`;
+    return this.getForeignKeysQuery({ tableName, schema: schemaName });
   }
 
   getForeignKeyReferenceQuery(table, columnName) {
```

Comment:
This method is never used and can be removed

Reply:
It's only used in the `getForeignKeyReferencesForTable` in the QueryInterface.

Reply:
This specific `getForeignKeyReferenceQuery` method, I can't find any reference to it anywhere in the codebase